src/main/java/com/iu/kmi/database/repository/AuftragRepository.java

```diff
@@ -0,0 +1,6 @@
+package com.iu.kmi.database.repository;
```

Comment:
Falsches package -> gehört in com.iu.kmi.repository oder com.iu.kmi.repositories